### ISSUE TITLE
Display PCMO (approver) Comments in Order Details View

### ISSUE THREAD

**Issue by choquette33** [peacecorps/medlink-android#18]
When a Staff member approves or denies an item or order, they have the ability to add "Extra Information" into the text box below the itemized order. The expectation is that the user will see this information when they view their processed orders. 

Currently only the "special instructions" that were entered by the user when placing the order are displayed. We need to show them both on the Order Details view. 

A note on the UI
I suggest keeping the "special instructions" displayed above the list of items, but I would like to see this  aligned right and displayed in a thought bubble shape. I would add the "extra information" to the bottom of the list of items included in that order, align it left and place it in a different colored thought bubble. 

I am attempting to recreate the UI of a chat (with out the avatar)
![image](https://cloud.githubusercontent.com/assets/2973990/14219333/aac9877a-f827-11e5-9133-bfa1906bba72.png)

**Comment by neerajrao** [comment]
@choquette33 right now the API does _not_ send back the extra information entered by the PCMO. I confirmed this by adding the extra information `Here is some extra information sent by the PCMO.` to the request created at `2016-04-04 00:57:01 -0400` by user `neeraj2608+2@gmail.com`. The extra information is displayed on the website:

![image](https://cloud.githubusercontent.com/assets/3719196/14238553/9939668a-fa00-11e5-8f5d-e312747367cb.png)

but does NOT appear in the API response:

```
    {
      "text": null,
      "created_at": "2016-04-04T00:57:01.073-04:00",
      "supplies": [
        {
          "id": 6,
          "response": {
            "id": 1937,
            "created_at": "2016-04-04T00:57:25.399-04:00",
            "type": "Delivery"
          },
          "status": "responded",
          "responded_at": "2016-04-04T00:57:25.399-04:00"
        }
      ]
    }
```

Could you please file an improvement in the `medlink` repo and add a link here?

**Comment by jamesdabbs** [comment]
Hey, @neerajrao - adding that's no problem, but I do have a design question. Recall that the various supplies may have different responses, and that a response may apply to some - but not all - supplies from one request. Given that, where would you prefer to get the extra text attached to a response?

1) Inline and duplicated inside the existing responses `{... "type": "Delivery", "extra_information": "Here is some ..." }`
2) As another top-level key `{... "supplies": [...], "responses": { "1937": { "created_at": ..., "extra_information": ... }}}`
3) From another endpoint `/api/v1/responses/1937` => `{ "created_at": ..., "extra_information": ... }`

**Comment by neerajrao** [comment]
@jamesdabbs good point. I think inline it with each supply's response. That way, if the extra info is different for each supply, we can still account for it.

@choquette33 given what James said, we might need to rethink how we're showing the extra information. I think you suggested showing it as a darker chat bubble at the very end, below all the supplies but we may have different extra info for each supply. Maybe add a bubble beneath each supply route?

**Comment by choquette33** [comment]
@neerajrao I wouldn't want repetitive text bubbles. Can we group the items with the same extra information in a list and only show one bubble per unique extra information text?
